api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+{
+        "swagger": "2.0",
+        "info": {
+                "title": "Cilium API",
+                "description": "Cilium",
+                "version": "unversioned"
+        },
+        "schemes": [
+                "http"
+	],
+        "x-schemes": [
+		"unix"
+        ],
+        "basePath": "/v1",
+        "produces": [
+                "application/json"
+        ],
+        "consumes": [
+                "application/json"
+        ],
+        "paths": {
+                "/healthz": {
+                        "get": {
+                                "summary": "Get health of Cilium daemon",
+                                "description": "Represents the runtime status of the daemon and related components.",
+				"tags": [ "daemon" ],
+                                "responses": {
+                                        "200": {
+                                                "description": "Success",
+                                                "schema": {
+                                                        "$ref": "#/definitions/StatusResponse"
+                                                }
+                                        }
+                                }
+                        }
+                },
+                "/config": {
```

Comment:
Fair question. The question is whether that would apply to `/ipam/` and `/policy/` as well then.